nestkernel/node_collection.h

```diff
@@ -363,6 +351,13 @@ class NodeCollection
    */
   virtual long find( const index ) const = 0;
 
+  /**
+   * Returns whether the NodeCollection has contains any nodes with proxies or not.
```

Comment:
Just for clarification: because of requirements on NCs with metadata, either all or none of the nodes in a NC have proxies, right? We should make that clear in the comment.